guides/_posts/2015-02-05-layout.html

```diff
@@ -205,34 +205,34 @@
 
 ## Intrinsic values
 
-Positioning elements with `top` and `left` properties doesn't require GSS to know the dimensions of that elements. But `right` and `bottom` properties need to know about `width` and `height` of the element respectively. Properly constrained system will provide those values indirectly with no regard to elements contents.
+Positioning elements with `top` and `left` properties doesn't require GSS to know the dimensions of those elements. But `right` and `bottom` properties need to know about `width` and `height` of the element respectively. Properly constrained systems will provide those values indirectly with no regard to element contents.
 
-However it's often desirable to know the computed dimensions of specific elements which GSS has no control over - be it text, images or blocks styled with CSS. Measurements provide dynamic "islands" to the layout, which can be automatically recomputed, whenever measured things are updated from the outside. It also may act as a feedback to the solver, disallowing content to shrink too much or helping to decide if things fit.
+However, it's often desirable to know the computed dimensions of specific elements which GSS has no control over - be it text, images or blocks styled with CSS. Measurements provide dynamic "islands" to the layout, which can be automatically recomputed, whenever measured things are updated from the outside. It also may act as a feedback to the solver, disallowing content to shrink too much or helping to decide if things fit.
 
-Prepending property with `intrinsic-` or `computed-` prefix makes GSS measure the values.
+Prepending a property with `intrinsic-` or `computed-` prefix makes GSS measure the values.
```

Comment:
Yes, that can work, adjusted to singular "value" instead of "values". There's some duplication with upcoming statements, though.

I'm not sure if the second part of the suggestion applies. I'm only just wrapping my head around GSS so I don't know if making properties read-only applies for both prefixes.

This whole section on `intrinsic` and `computed` needs more clarification, but I can't provide it at the moment. I'm lacking full understanding.